towhee/pipeline/pipeline.py

```diff
@@ -0,0 +1,296 @@
+from typing import Callable
+from towhee.pipeline.action import Action
+import uuid
+import json
+
+# pylint: disable=protected-access
+# pylint: disable=redefined-builtin
+
+
+class Pipeline:
```

Comment:
I think we need to add `tag` key to the dag, which defaults to 'main'.

Reply:
The issue came up of how does the user input that data. Ops are sepearte from pipeline. And currently ops are created with ops.repo.name(args, kwargs). One option is to do ops.repo.name['tag'](args, kwargs), but that is pretty ugly.